src/storage/mvcc/reader/point_getter.rs

```diff
@@ -186,7 +191,7 @@ impl<S: Snapshot> PointGetter<S> {
     ///
     /// First, a correct version info in the Write CF will be sought. Then, value will be loaded
     /// from Default CF if necessary.
-    fn load_data(&mut self, user_key: &Key) -> Result<Option<Value>> {
+    fn load_data(&mut self, user_key: &Key, can_be_cached: &mut bool) -> Result<Option<Value>> {
         if !self.write_cursor.seek(
             &user_key.clone().append_ts(self.ts),
```

Comment:
Here we directly seek to `self.ts`, then it's impossible to see the higher versions.

Reply:
Fixed.
My rust code is horrible, thanks a lot for helping